packages/shared/src/styles/themes.scss

```diff
@@ -24,6 +24,7 @@
         --general-main-2: #{$color-white};
         --general-section-1: #{$color-grey-2};
         --general-section-2: #{$color-grey-2};
+        --general-section-3: #{$color-grey-4};
```

Comment:
Was this specification added to the Colour Palette style guide? (Same for dark theme.)

Reply:
Per our discussion, replaced with `general-main-2`.